api/app/src/routes/medical/organization.ts

```diff
@@ -14,9 +14,13 @@ import cwCommands from "../../external/commonwell";
 import { asyncHandler, getCxIdOrFail, getETag, getFromParamsOrFail } from "../util";
 import { dtoFromModel } from "./dtos/organizationDTO";
 import { organizationCreateSchema, organizationUpdateSchema } from "./schemas/organization";
+import { analytics, EventTypes } from "../../shared/analytics";
+import { ApiTypes } from "../../command/usage/report-usage";
 
 const router = Router();
 
+const ORG_PATH = "/organization";
```

Comment:
Ideally would get this from `req.baseUrl` - taking into account it can have more than one level.